code/__DEFINES/~pariah_defines/DNA.dm

```diff
@@ -17,3 +17,7 @@
 
 #define MANDATORY_FEATURE_LIST list("flavor_text" = "")
 
+#define ORGAN_SLOT_EXTERNAL_HEADTAILS "headtails"
```

Comment:
Is this actually how TG does external organs..? Genuine question.

Reply:
Seems so